charts/quantum-serverless/charts/repository/values.yaml

```diff
@@ -6,6 +6,10 @@ replicaCount: 1
 
 application:
   debug: 0
+  superuser:
```

Comment:
We may need to address [this issue](https://github.com/Qiskit-Extensions/quantum-serverless/issues/371).   Are we going to bump up to high priority?

Reply:
I think we should, I'm going to start with the integration with the infrastructure team so I think it has a lot of sense to start working on this.